src/libfsm/minimise.c

```diff
@@ -256,7 +278,7 @@ build_minimised_mapping(const struct fsm *fsm,
 				}
 			}
 
-			uint64_t checked_labels[4] = {0};
+			uint64_t checked_labels[256/64] = {0};
```

Comment:
what's happening here?

Reply:
It's called out in one of the commit logs but I didn't include it in the PR description:

>  change `checked_labels`'s length to `[256/64]` to be consistent with its definition elsewhere.

`256/64` us used in a bunch of other places for a character set of `FSM_SIGMA_COUNT` represented as a `uint64_t` bitset, and since you caught that typo before of `256/4` I wanted to make sure that was consistent.

Reply:
Oh! I actually misread this, that makes even more sense. I thought you were changing the array size here, but you're not.